client/src/pages/Communities/List/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'wouter'
+import { Button, Typography } from '@mui/material'
+
+const Page = styled.main`
+  width: 100%;
+  padding: 0.5rem;
+  margin-top: 5rem;
+`
+
+const ListCommunities = () => (
+  <Page>
```

Comment:
TODO: get a list of

- communities that the user is a part of
- public communities that the user might want to join

and display above the `create` cta